.github/workflows/ClimaCoreMakie.yml

```diff
@@ -15,12 +15,13 @@ jobs:
   lib-climacore-makie:
     runs-on: ubuntu-20.04
     timeout-minutes: 45
+    strategy:
+      matrix:
+        version: ['1.10', '1.11']
     steps:
       - name: Checkout
         uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v2
```

Comment:
You need this to:
```
        with:
          version: ${{ matrix.version }}
```

Same for the other github workflows below where you changed from 1.10 to 1.11